### DISCUSSION TITLE
There is any way to insert or ignore data with object?

### DISCUSSION THREAD

**Discussion by nmgwddj** [fnc12/sqlite_orm#985] [General]
Like normal insert:

```
storage_->insert(message);
```

I expected that when using insert or ignore I could specify an object instead of writing all the fields, like:

```
storage_->insert(or_ignore(), into<MessageLog>(), myobject);
```

**Comment by fnc12** [comment]
no there is not way of doing this. Either use simplified CRUD API with an object or non-CRUD detailed API with raw fields. BTW we can think about such API to implement it in future but right now it doesn't exist
